apps/api/src/app/analytics/usecases/hubspot-identify-form/hubspot-identify-form.usecase.ts

```diff
@@ -18,6 +18,7 @@ export class HubspotIdentifyFormUsecase {
         { name: 'email', value: command.email },
         { name: 'lastname', value: command.lastName || 'Unknown' },
         { name: 'firstname', value: command.firstName || 'Unknown' },
+        { name: 'app_organizationid', value: command.organizationId },
```

Comment:
This passes the current active org as an optional param to hubspot, can help us with matching user to account at later stage,